cmd/internal/flowmode/cmd_run.go

```diff
@@ -102,6 +102,8 @@ depending on the nature of the reload error.
 		StringVar(&r.clusterJoinAddr, "cluster.join-addresses", r.clusterJoinAddr, "Comma-separated list of addresses to join the cluster at")
 	cmd.Flags().
 		StringVar(&r.clusterDiscoverPeers, "cluster.discover-peers", r.clusterDiscoverPeers, "List of key-value tuples for discovering peers")
+	cmd.Flags().
+		StringSliceVar(&r.clusterAdvInterfaces, "cluster.advertise-interfaces", r.clusterAdvInterfaces, "List of interfaces used to infer an address to advertise")
```

Comment:
nit: Can we set the default right on the `flowRun` struct on line 42 instead of implicitly setting it on the code later on?

This way the default value will still be available in case we use it elsewhere and can just call `opts.AdvertiseInterfaces` directly as an argument to FirstAddress.